src/components/app/index.js

```diff
@@ -23,9 +23,11 @@ const App = () => (
       <PrivateRoutes path='/main' render={ ({ match }) => (
         <main>
           <Account />
-          <Link to={`${match.url}/transactions`}>Transactions</Link>
-          <Link to={`${match.url}/voting`}>Voting</Link>
-          <Link to={`${match.url}/forging`}>Forging</Link>
+          <section className='main-tabs'>
```

Comment:
Good point. But not worth fixing because it will be replaced with `<Tabs>` in https://github.com/LiskHQ/lisk-nano/issues/505 anyway.